spec/inquisition/rubocop/runner_spec.rb

```diff
@@ -0,0 +1,56 @@
+RSpec.describe Inquisition::RuboCop::Runner do
+  let(:runner) { described_class.new }
+  let(:message) { 'Warn message' }
+  let(:file) { 'foo/bar.rb' }
+
+  let(:convention_severity) { instance_double(RuboCop::Cop::Severity, name: :convention) }
+  let(:warning_severity)    { instance_double(RuboCop::Cop::Severity, name: :warning) }
+  let(:error_severity)      { instance_double(RuboCop::Cop::Severity, name: :error) }
+
+  let(:convention_offense) do
+    instance_double(RuboCop::Cop::Offense, severity: convention_severity, message: message, line: nil)
+  end
+
+  let(:warning_offense) do
+    instance_double(RuboCop::Cop::Offense, severity: warning_severity, message: message, line: nil)
+  end
+
+  let(:error_offense) do
+    instance_double(RuboCop::Cop::Offense, severity: error_severity, message: message, line: nil)
+  end
+
+  let(:offenses) { [[file, [convention_offense]], [file, [warning_offense]], [file, [error_offense]]] }
+
+  let(:high_level_issue) do
```

Comment:
you dont need to create objects since, you can't check their equality, you can just use hashes with values you'll need to match